src/model/mod.rs

```diff
@@ -21,24 +22,26 @@ pub struct Block {
     pub timestamp: u64,
     /// array of transactions, which successfully passed validation and consensus step.
     pub transactions: Vec<Transaction>,
-    /// hash of a previous block in the chain.
-    //TODO[@humb1t:RH2-9]: what to do if this block first?
-    pub previous_block_hash: Hash,
+    /// Hash of a previous block in the chain.
+    /// Is None only for the first block.
+    pub previous_block_hash: Option<Hash>,
```

Comment:
Can we remove optional here? Every block will have this field. Very first block will have it filled with 0s

Reply:
Ok, will fix that